aten/src/ATen/native/native_functions.yaml

```diff
@@ -5114,13 +5114,13 @@
 - func: quantize_per_channel(Tensor self, Tensor scales, Tensor zero_points, int axis, ScalarType dtype) -> Tensor
   variants: function
   dispatch:
-    CPU, CUDA: quantize_per_channel_cpu
+    CPU, CUDA: quantize_per_channel
```

Comment:
Are there any BC concerns with re-naming this operator?

Reply:
Not sure if we have any use-cases where we directly call `at::native::quantize_per_channel_cpu`

Reply:
I was under the impression that anything external would just go through the dispatcher.

Reply:
perhaps @raghuramank100 or @vkuzo would have a better idea

Reply:
Doing fbgs shows only internal usage within torch : https://www.internalfb.com/intern/codesearch/?bunny_arg&bunny_command=fbgs&lucky=0&q=repo%3Afbcode%20case%3Ainsensitive%20regex%3Aoff%20quantize_per_channel_cpu&source=redirect
so this is ok